docs/collection_metadata.rst

```diff
@@ -1,66 +1,46 @@
    collection
--  A file named ``bindep.txt`` is in the root level of the collection
+-  A file named ``bindep.txt`` in the root level of the collection
 
-If any of these files are in the ``build_ignore`` of the collection, it
-will not work correctly.
+These files must be included in the packaged collection on Galaxy. Ansible Builder cannot install dependencies listed in files that are included in the ``build_ignore`` of a collection, because those files are not uploaded to Galaxy.
 
 Collection maintainers can verify that ``ansible-builder`` recognizes
-the requirements they expect by using the ``introspect`` command. Example:
+the requirements they expect by using the ``introspect`` command. For example:
 
-::
+.. code-block:: text
 
     ansible-builder introspect --sanitize ~/.ansible/collections/
 
-The ``--sanitize`` option will go through all of the collection requirements and
-remove duplicates, as well as remove some Python requirements that should normally
-be excluded (see :ref:`python_deps` below).
+The ``--sanitize`` option reviews all of the collection requirements and removes duplicates. It also removes any Python requirements that should normally be excluded (see :ref:`python_deps` below).
 
 .. note::
-    Use the ``-v3`` option to ``introspect`` to see logging messages about requirements
-    that are being excluded.
+    Use the ``-v3`` option to ``introspect`` to see logging messages about requirements that are being excluded.
 
 .. _python_deps:
 
 Python Dependencies
 ^^^^^^^^^^^^^^^^^^^
 
-Python requirements files are combined into a single file using the
-``requirements-parser`` library in order to support complex syntax like
-references to other files.
+Ansible Builder combines all the Python requirements files from all collections into a single file using the ``requirements-parser`` library. This library supports complex syntax, including references to other files.
 
-Entries from separate collections that give the same *package name* will
-be combined into the same entry, with the constraints combined.
+If multiple collections require the same *package name*, Ansible Builder combines them into a single entry and combines the constraints.
 
-There are several package names which are specifically *ignored* by
-``ansible-builder``, meaning that if a collection lists these, they will
-not be included in the combined file. These include test packages and
-packages that provide Ansible itself. The full list can be found in
-``EXCLUDE_REQUIREMENTS`` in the ``ansible_builder.requirements`` module.
+Certain package names are specifically *ignored* by ``ansible-builder``, meaning that Ansible Builder does not include them in the combined file of Python dependencies, even if a collection lists them as dependencies. These include test packages and packages that provide Ansible itself. The full list can be found in ``EXCLUDE_REQUIREMENTS`` in the ``ansible_builder.requirements`` module.
```

Comment:
Is there a link or something for the `ansible_builder.requirements` module or am I missing something obvious here? I tried searching for it but can't find it...

Reply:
Oh, nice catch! I just changed the formatting without changing the content here. I see the list in `src/ansible_builder/_target_scripts/introspect.py`. I'll update the text and add a link in my next changeset.